src/provider.js

```diff
@@ -448,16 +446,27 @@ export default function () {
         // redirect to login page
         $timeout(() => {
           const params = [];
+          let destUrl = signUpUrl;
+          const hasHashInSignUpUrl = destUrl.indexOf('#') > -1;
+          let urlPart = destUrl;
+
+          if (hasHashInSignUpUrl) {
+            const splittedUrl = signUpUrl.split('#');
+            const hashPartIndex = 1;
+            urlPart = splittedUrl[hashPartIndex];
+          } else {
+            destUrl += '#/';
+          }
 
-          if (signUpUrl.indexOf('onsuccess') === -1) {
+          if (urlPart.indexOf('onsuccess') === -1) {
```

Comment:
use bitwise instead ? :D

Reply:
Not aligned with the actual ESLint configuration :D